lib/generators/react_on_rails/generator_helper.rb

```diff
@@ -17,6 +17,16 @@ def puts_setup_file_error(file, data)
     puts "\n#{data}\n"
   end
 
+  def return_setup_file_error(file, data)
+    data = <<-DATA.strip_heredoc
```

Comment:
@aaronvb We ran into this a lot. I prefer to just skip the `strip_heredoc` and remove the leading spaces.